napalm_ios/ios.py

```diff
@@ -142,23 +145,66 @@ def is_alive(self):
             'is_alive': self.device.remote_conn.transport.is_active()
         }
 
+    @staticmethod
+    def _create_tmp_file(config):
+        """Write temp file and for use with inline config and SCP."""
+        tmp_dir = os.getcwd()
```

Comment:
I remember I had issues in the past assuming the user has rights to create files in the CWD.
Would be possible to change this to the temp dir? If we don't want to hardcode it to `/tmp`, we can do the following (cross-platform):

```python
import tempfile
tempfile.gettempdir()
```

Would you agree?